generate.js

```diff
@@ -60,8 +63,45 @@ const template = `
         </div>
       {{/orgs}}
     </div>
+    <script>
+      document.getElementById("last-generated").textContent = "Last generated " + timeDifference(new Date().getTime(), new Date('{{timestamp}}').getTime())
+
+      function timeDifference(current, previous) {
+        const msPerMinute = 60 * 1000;
+        const msPerHour = msPerMinute * 60;
+        const msPerDay = msPerHour * 24;
+        const msPerMonth = msPerDay * 30;
+        const msPerYear = msPerDay * 365;
+        
+        const elapsed = current - previous;
+        
+        if (elapsed < msPerMinute) {
+          return Math.round(elapsed/1000) + ' seconds ago';   
+        }
+        
+        else if (elapsed < msPerHour) {
+          return Math.round(elapsed/msPerMinute) + ' minutes ago';   
+        }
+        
+        else if (elapsed < msPerDay ) {
+          return Math.round(elapsed/msPerHour ) + ' hours ago';   
+        }
+        
+        else if (elapsed < msPerMonth) {
+          return 'approximately ' + Math.round(elapsed/msPerDay) + ' days ago';   
+        }
+        
+        else if (elapsed < msPerYear) {
+          return 'approximately ' + Math.round(elapsed/msPerMonth) + ' months ago';   
+        }
+        
+        else {
+          return 'approximately ' + Math.round(elapsed/msPerYear ) + ' years ago';   
+        }
+      }
```

Comment:
Just a heads-up, this repo will use prettier.io code formatter via ESLintBear.

Ideally you should start formatting it now (using `prettier` directly)

Reply:
do I need to prettify with this https://prettier.io/playground/ ?

Reply:
Sure that'll work for now.

Reply:
Don't forget to set it without semicolon though :)

Reply:
And with trailing comma